File: python/raindrops/raindrops.py
def convert(number):
    """
    Convert a number to a raindrop sound.
    If the number doesn't have any of the numbers in the dictionary as a factor
    then return the number (as a string). Otherwise concatenate all of the 
    values for matching factors together and return that.
    """
    lookup = {
        3: 'Pling',
        5: 'Plang',
        7: 'Plong'
    }

    ret = ''
    for key, value in lookup.items():
        if number % key == 0:
            ret += value
    
    if ret == '':
        ret = str(number)
    
    return ret

File: php/raindrops/Raindrops.php
<?php
// Raindrops exercise

declare(strict_types=1);

// Return a raindrop sound for a given number.
// @param $number: The number to convert to a raindrop sound
// @returns: The text of the raindrop sound.
function raindrops(int $number): string
{
    $result = "";

    if ($number % 3 == 0) {
        $result .= "Pling";        
    }
    if ($number % 5 == 0) {
        $result .= "Plang";
    }
    if ($number % 7 == 0) {
        $result .= "Plong";
    }
    return strlen($result) == 0 ? strval($number) : $result;
}
